crates/bitwarden-json/src/command.rs

```diff
@@ -188,3 +190,11 @@ pub enum GeneratorsCommand {
     /// Returns: [String]
     GeneratePassword(PasswordGeneratorRequest),
 }
+
+#[cfg(debug_assertions)]
+#[derive(Serialize, Deserialize, JsonSchema, Debug)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub enum DebugCommand {
+    CancellationTest { duration_millis: u64 },
+    ErrorTest {},
+}
```

Comment:
An alternative way. of doing this would be to add one or two separate commands to bitwarden-c.